actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -312,9 +300,9 @@ def expires_now
     # You can use this method when you have an HTTP response that never changes, and
     # the browser and proxies should cache it indefinitely.
     #
-    # *   `public`: By default, HTTP responses are private, cached only on the
-    #     user's web browser. To allow proxies to cache the response, set `true` to
-    #     indicate that they can serve the cached response to all users.
+    # `public`: By default, HTTP responses are private, cached only on the
+    #  user's web browser. To allow proxies to cache the response, set `true` to
+    #  indicate that they can serve the cached response to all users.
```

Comment:
I'd prefer we make this use the "correct" definition list syntax as my other comment since it also isn't multi paragraph